pkg/manifest/installer.go

```diff
@@ -285,7 +285,11 @@ func applyManifest(componentName name.ComponentName, manifestStr string, version
 
 	appliedObjects = append(appliedObjects, objects...)
 
-	extraArgs := []string{"--force", "--prune", "--selector", fmt.Sprintf("%s=%s", istioComponentLabelStr, componentName)}
+	extraArgs := []string{"--force"}
+	// Base components include namespaces and CRDs, do not prune them.
+	if componentName != name.IstioBaseComponentName {
```

Comment:
i agree we don't want to prune them but pls leave a comment that pruning crds removes user config, which makes it impossible to roll back.